bootstrap/test/test-clean.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2017 The Absolute Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+const spawnSync = require('child_process').spawnSync;
+
+describe('Clean Test', () => {
+  it('Is out directory cleaned?', () => {
+    spawnSync(path.resolve('./absolute'), ['start']);
```

Comment:
Let's improve this test.
1. Check whether |out| directory exists or not. (including  .pkg_timestamp) - ASSERT not exists
2. Run absolute build
3. Check whether |out| directory exists or not - ASSERT exists
4. Run absolute clean
5. Check whether |out| directory exists or not - ASSERT not exists again.